include/el/msglink/networking_architecture.md

```diff
@@ -86,59 +88,59 @@ For the communication to work, both parties need to have a compatible link. The
 
-At it's core, msglink is a point-to-point communication protocol. msglink never uses broadcasts or multicasts on a network level. There are always two parties A and B communication with each other using the msglink protocol.
+At it's core, msglink is a point-to-point communication protocol. msglink never uses broadcasts or multicasts on a network level. There are always exactly two parties A and B communication with each other using the msglink protocol.
 
-In the msglink protocol, there is no logical difference between a client and a server. Both communication parties are equal and have equal capabilities (in the bounds defined by the user application).
+In the msglink protocol, there is no logical difference between a client and a server. Both communication parties are equal and have equal capabilities (within the bounds defined by the user application).
 
-However, at some point, a network connection (Websocket, which builds ontop of TCP socket) needs to be established as a communication channel. For that to work, one of the two parties needs to act as a server, listening for a new connection by the other party. It is the job of the **Network Interface** architecture block to manage this connection phase. This network interface block is typically represented by a class. There are multiple network interface blocks that the user of the library can select from to assign a communication party the appropriate role in the connection establishing phase.
+However, at some point, a network connection (websocket, which builds ontop of TCP socket) needs to be established as a communication channel. For that to work, one of the two parties needs to act as a server, listening for a new connection by the other party. It is the job of the **Network Interface** architecture block to manage this connection phase. This network interface block is typically represented by a class. There are multiple network interface blocks the user of the library can select from to assign a communication party the appropriate role in the connection-establishing phase.
 
 ### Networking role selection
 
 Depending on the application, the user may arbitrarily define either party to be the server or the client by using the corresponding network interface class ```el::msglink::server``` or ```el::msglink::client```.
 
-In many cases, like a WebApp, the role selection is obvious. In this example, it only makes sense for a program running on the webserver with a public facing IP address to be the msglink server and the browser to be the client.<br>
-In most applications, one party is clearly the "boss" of operations and it therefor makes sense for that one to be the server.
+In many cases (e.g. like a WebApp), the role selection is obvious. In this example, it only makes sense for a program running on the webserver with a public facing IP address to be the msglink server and the browser to be the client.<br>
```

Comment:
Cannot be in brackets because following sentence references it.

"this" needs to be removed after "e.g.".

Reply:
Accepted!